packages/foam-vscode/src/features/preview/wikilink-embed.ts

```diff
@@ -47,25 +47,20 @@ export const markdownItWikilinkEmbed = (
         let content = `Embed for [[${wikilink}]]`;
         switch (includedNote.type) {
           case 'note': {
-            let noteText = readFileSync(includedNote.uri.toFsPath()).toString();
-            const section = Resource.findSection(
+            const noteStyle = getFoamVsCodeConfig(
```

Comment:
In the next phase, I'd update the regex to capture the new syntax. Then here we'd have logic to determine `noteStyle` and `noteContent`. For now, `noteStyle` pulls from the existing config setting, and there is no `noteContent`, it just defaults to "full".

Reply:
That's correct, although from the configuration we should be getting both the style and the content, is that planned for this PR?

Reply:
So I'm a little fuzzy on release strategy. The new `preview.embedStyle` isn't used anywhere at the moment, but I can change it so that this wikilink-embed reads it and it overrides `preview.embedNoteInContainer` if both are set. I can also add a deprecation flag to  `preview.embedNoteInContainer` so it no longer appears in settings UI. In the next PR, I can remove it altogether `package.json` as well as this file. wdyt?

Reply:
The release strategy you outlined is what I had in mind and I think works pretty well.

I am not aware of a built-in way in VS Code to do that, did you come across something or had something in mind?

Reply:
Do you mean for the deprecation flag? https://code.visualstudio.com/api/references/contribution-points#contributes.configuration:~:text=deprecationMessage%20/%20markdownDeprecationMessage

Reply:
Yes, that's great, I was already thinking of a small utility to deal with deprecation, but that won't be necessary 👍

Reply:
sweet!